service/src/main/kotlin/no/nav/su/se/bakover/service/revurdering/RevurderingServiceImpl.kt

```diff
@@ -590,16 +612,20 @@ internal class RevurderingServiceImpl(
             InformasjonSomRevurderes.tryCreate(oppdaterRevurderingRequest.informasjonSomRevurderes)
                 .getOrHandle { return KunneIkkeOppdatereRevurdering.MåVelgeInformasjonSomSkalRevurderes.left() }
 
-        val gjeldendeVedtaksdata = vedtakService.kopierGjeldendeVedtaksdata(
-            sakId = revurdering.sakId,
+        val sak = sakService.hentSakForRevurdering(oppdaterRevurderingRequest.revurderingId)
```

Comment:
Kanskje ikke nødvendig å hente revurderingen fra databasen noen linjer opp.

Reply:
Så det, men var redd for at det skulle sende meg inn i en vond spiral med endringer av tester. Gir det et forsøk!